lib/stores/BaseStore.js

```diff
@@ -63,6 +63,17 @@ export default class BaseStore extends Flux.MapStore {
   }
 
   /**
+   * Returns the count for the given query
+   *
+   * @param {ResourceQuery} [query]
+   *
+   * @return {number}
+   */
+  count(query) {
```

Comment:
With resources this is an opt-in feature. It should be the same for stores, I think